app/controllers/monthly_reports_controller.rb

```diff
@@ -1,6 +1,20 @@
 class MonthlyReportsController < ApplicationController
   def index
-    @monthly_reports = MonthlyReport.all
+    @monthly_reports = MonthlyReport.page params[:page]
+  end
+
+  def mine
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for mine is too high. [22.93/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)